crypto/x509/standard_exts.h

```diff
@@ -81,6 +82,7 @@ static const X509V3_EXT_METHOD *standard_exts[] = {
     &ossl_v3_single_use,
     &ossl_v3_group_ac,
     &ossl_v3_holder_name_constraints,
+    &ossl_v3_associated_info,
```

Comment:
last comma is useless

Reply:
This is just a personal preference, to some extent, but the reason I prefer the trailing comma is that it makes version control diffs smaller and simpler: https://eslint.org/docs/latest/rules/comma-dangle.

Reply:
This comma creates an ending <<nul>> item, who is useless.

Reply:
> This comma creates an ending << nul >> item, who is useless.

This is not correct. There is no extra array item created by this trailing comma.